package.json

```diff
@@ -4,7 +4,7 @@
     "ansi-regex": "^3.0.0",
     "ansi-styles": "^3.2.0",
     "babel-core": "^6.23.1",
-    "babel-eslint": "^7.2.3",
+    "babel-eslint": "^8.2.3",
```

Comment:
do we not get a peer dep warning from this?

https://github.com/facebook/fbjs/pull/275 hasn't been released yet, has it?

Reply:
yup, we do, missed that. @cpojer might push someone to actually make a release? :p in the meantime I'll revert this change

Reply:
you'll need to downgrade eslint as well

Reply:
oh man